Modules/IO/MeshVTK/test/itkPolylineCellTest.cxx

```diff
@@ -0,0 +1,58 @@
+/*=========================================================================
+ *
+ *  Copyright NumFOCUS
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *=========================================================================*/
+
+#include "itkMesh.h"
+#include "itkMeshFileTestHelper.h"
+#include "itkTestingMacros.h"
+
+int
+itkPolylineCellTest(int argc, char * argv[])
+{
+  if (argc < 3)
+  {
+    std::cerr << "Invalid commands, You need input and output mesh file name " << std::endl;
+    return EXIT_FAILURE;
+  }
+
+  bool IsBinary = (argc > 3);
+
+  constexpr unsigned int dimension = 3;
+  using PixelType = itk::VariableLengthVector<float>;
+  using MeshType = itk::Mesh<PixelType, dimension>;
+  using CellTraits = MeshType::CellTraits;
+  using CellInterfaceType = itk::CellInterface<PixelType, CellTraits>;
+  using PolyLineCellType = itk::PolyLineCell<CellInterfaceType>;
+
+  // Test instantiating cell and exercise basic methods.
+  auto testCell = new PolyLineCellType;
+
+  // List the points that the polyline cell will use.
+  MeshType::PointIdentifier polyLinePoints[5] = { 0, 1, 2, 3, 4 };
+
+  ITK_TRY_EXPECT_NO_EXCEPTION(testCell->SetPointIds(polyLinePoints));
+  ITK_TEST_EXPECT_TRUE(testCell->GetNumberOfVertices() == 5);
+  ITK_TRY_EXPECT_NO_EXCEPTION(testCell->PointIdsBegin());
+  ITK_TRY_EXPECT_NO_EXCEPTION(testCell->PointIdsEnd());
+
```

Comment:
Can all public methods be exercised, please?

Reply:
I just discovered that we have an [itkCellInterfaceTest](https://github.com/InsightSoftwareConsortium/ITK/blob/971aef211cad336ba75cb2da114fc86e873444e5/Modules/Core/Mesh/test/itkCellInterfaceTest.cxx) so I will use that instead of performing tests here. Also
that code tests all the methods so the code coverage will be fully covered.